flake8_quotes/__init__.py

```diff
@@ -163,11 +191,22 @@ def get_quotes_errors(self, file_contents):
             #   "foo"[0] * 3 = " * 3 = """
             #   "foo"[0:3] = "fo
             #   """foo"""[0:3] = """
+            is_docstring = token in docstring_tokens
             is_multiline_string = unprefixed_string[0] * 3 == unprefixed_string[0:3]
             start_row, start_col = token.start
 
+            # If our string is a docstring
+            if is_docstring:
```

Comment:
We should prob add a comment about how `is_docstring` **must** come before `is_multiline_string` as docstrings are a subset. I can take care of this when landing the PR though